moveit_core/setup.py

```diff
@@ -0,0 +1,11 @@
+## ! DO NOT MANUALLY INVOKE THIS setup.py, USE CATKIN INSTEAD
+
+from distutils.core import setup
+from catkin_pkg.python_setup import generate_distutils_setup
+
+d = generate_distutils_setup(
+    packages=['moveit_core'],
+    package_dir={'': 'planning_scene'}
```

Comment:
Why are you referring to `planning_scene` folder here?

Reply:
simple mistake, good catch!